week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//generate umber between 0 and 9
```

Comment:
A question to think about...

What `type` of value does this return? A `number` or a `float`? How can we be sure of it?

If it is a `float`, how can we turn this into a `number`? Think about the capabilities of the `Math` object and what we learnt in the last lesson.